src/components/views/verification/VerificationShowSas.js

```diff
@@ -116,9 +117,11 @@ export default class VerificationShowSas extends React.Component {
             let text;
             if (this.state.pending) {
                 if (this.props.isSelf) {
+                    // device shouldn't be null in this situation but it seems like it can be, so show
+                    // slightly broken text rather than soft-crashing (we've already logged in VerificationPanel).
                     text = _t("Waiting for your other session, %(deviceName)s (%(deviceId)s), to verify…", {
```

Comment:
this will look weird as `Waiting for your other session, (), to verify...`

Reply:
yeah, it will - think it's worth adding another translatable string to make the edge case not look weird?

Reply:
Yeah